examples/mnist-inference-web/index.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * 
+ * This demo is part of Burn project: https://github.com/burn-rs/burn
+ * 
+ * Released under a dual license: 
+ * https://github.com/burn-rs/burn/blob/main/LICENSE-MIT
+ * https://github.com/burn-rs/burn/blob/main/LICENSE-APACHE
+ * 
+ */
+
+/**
+ * Auto crops the image, scales to 28x28 pixel image, and returns as grayscale image.
+ * @param {object} mainContext - The 2d context of the source canvas.
+ * @param {object} cropContext - The 2d context of an intermediate hidden canvas.
+ * @param {object} scaledContext - The 2d context of the destination 28x28 canvas.
+ */
+export function cropScaleGetImageData(mainContext, cropContext, scaledContext) {
```

Comment:
It would have been be really cool to have those transformations made with rust, maybe in a future refactoring.

Reply:
Yes, absolutely. You can write the whole logic in Rust but I tried to make it simple.